docs/content/post/examples.md

```diff
@@ -0,0 +1,120 @@
++++
+author = "Daniel Mellado"
+title = "Examples"
+date = "2020-10-15"
+description = "Example configs for Egress Router CNI"
+tags = [
+    "cni",
+    "egress",
+    "developer"
+]
++++
+
+## Example configurations for both NetworkAttachmentDefinition and pod
+
+### NetworkAttachmentDefinition with json CNI config
+```yaml
+---
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+  name: egress-router-cni-net
+spec:
+  config: '{
+    "cniVersion": "0.4.0",
+    "type": "egress-router",
+    "name": "egress-router-cni-net",
+    "ip": {
+      "addresses": [
+        "192.168.123.99"
+        ],
+      "destinations": [
+        "192.168.123.91"
+      ],
+      "gateway": "192.168.123.1"
+      }
+    }'
+```
+
+### NetworkAttachmentDefinition with CNI config file
+If NetworkAttachmentDefinition has no spec, multus find a file in
+defaultConfDir (‘/etc/cni/multus/net.d’, with same name in the ‘name’ field of
+CNI config.
+```yaml
```

Comment:
The wording here is a bit confusing, do you mean "If the NetworkAttatchmentDefinition has no spec, multus will look for a file in the defaultConfDir (‘/etc/cni/multus/net.d’), with same name that's specified in the CNI config."